src/exceptions/PS-or-PDF-font-exception-20170817.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="PS-or-PDF-font-exception-20170817" name="PS/PDF font exception (2017-08-17)" listVersionAdded="3.2">
+      <crossRefs>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/master/LICENSE</crossRef>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/65962e27febc3883a17e651cdb23e783668c996f/LICENSE</crossRef>
+      </crossRefs>
+      <notes>Specified to be used with APGL-3.0</notes>
+    <text>
+      <optional>
+        <p>The font and related files in this directory are distributed under the
+        GNU AFFERO GENERAL PUBLIC LICENSE Version 3 (see the file COPYING), with
+        the following exemption:</p>
```

Comment:
I think this paragraph is part of the standard license header, and not part of the exception text itself.  Did the legal team have time to discuss [my proposal to allow `<text>` inside `<standardLicenseHeader>`][1]?  If not, do we want to take the less-DRY alternative I floated there?

[1]: https://github.com/spdx/license-list-XML/issues/616#issuecomment-382835783

Reply:
I think this is fine as is and consistent with the way we've been doing it. As for your proposal - we probably ought to slot that in for a discussion on an upcoming call